c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -1472,52 +1455,6 @@ void MfxC2DecoderComponent::DoUpdateMfxParam(const std::vector<C2Param*> &params
                 }
                 break;
             }
-            case kParamIndexColorAspects: {
-                android::ColorAspects ca;
```

Comment:
What if the app config color aspect? Did we handle this case?

Thanks,
Tianmi

Reply:
> What if the app config color aspect? Did we handle this case?
> 
> Thanks, Tianmi

yes, of course we can handle this through base class's function
https://github.com/projectceladon/MediaSDK_C2/blob/6459655c92f5737488844b06d9c9d2e8f65d3db2/c2_components/src/mfx_c2_component.cpp#L159

https://github.com/projectceladon/MediaSDK_C2/blob/6459655c92f5737488844b06d9c9d2e8f65d3db2/c2_components/src/mfx_c2_component.cpp#L146

to get/set the parameters by default

Reply:
Please share the detailed info where we can get and set the color aspects?

Thanks,
Tianmi

Reply:
> Please share the detailed info where we can get and set the color aspects?
> 
> Thanks, Tianmi

this is set:
https://github.com/projectceladon/MediaSDK_C2/blob/6459655c92f5737488844b06d9c9d2e8f65d3db2/c2_components/src/mfx_c2_decoder_component.cpp#L128

get is though query function from framework as it was registed here
https://github.com/projectceladon/MediaSDK_C2/blob/6459655c92f5737488844b06d9c9d2e8f65d3db2/c2_components/src/mfx_c2_decoder_component.cpp#L591